python/ray/includes/libcoreworker.pxd

```diff
@@ -189,6 +190,8 @@ cdef extern from "ray/core_worker/core_worker.h" nogil:
         CRayStatus SetResource(const c_string &resource_name,
                                const double capacity,
                                const CClientID &client_Id)
+        CRayStatus SpillObjects(const c_vector[CObjectID] &object_ids)
+        CRayStatus RestoreSpilledObjects(const c_vector[CObjectID] &object_ids)
```

Comment:
These RPCs are both for the experimental API only right? Can you call them `ForceSpillObjects` and `ForceRestoreObjects` for clarity then?

Reply:
sure.